dsp/modules/__init__.py

```diff
@@ -8,4 +8,5 @@
 from .pyserini import *
 
 from .hf_client import HFClientTGI
-from .hf_client import Anyscale
\ No newline at end of file
+from .llama import LlamaCpp
```

Comment:
Hmm importing this by default means that people need a new dependency right? It should be optional